<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InteractivePatchComparisonWidget</class>
 <widget class="QMainWindow" name="InteractivePatchComparisonWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>841</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Interactive Patch Comparison</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,3,1">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0,0,0,0,0,0">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Patch Radius</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="txtPatchRadius">
            <property name="toolTip">
             <string>Sets the width/height of a square patch</string>
            </property>
            <property name="text">
             <string>15</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>&lt;font color=&quot;green&quot;&gt;Source Patch&lt;/font&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PatchInfoWidget" name="SourcePatchInfoWidget" native="true"/>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>&lt;font color=&quot;red&quot;&gt;Target Patch&lt;/font&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="PatchInfoWidget" name="TargetPatchInfoWidget" native="true"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Sum Abs.Pixel Diff:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblSumAbsolutePixelDifference">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Correlation:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="lblCorrelation">
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="btnSavePatches">
          <property name="text">
           <string>Save Patches</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QVTKWidget" name="qvtkWidget"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2"/>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLeft_Pane">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenImage"/>
    <addaction name="actionOpenMask"/>
    <addaction name="actionOpenMaskInverted"/>
    <addaction name="actionFlipImage"/>
    <addaction name="actionQuit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <addaction name="menuLeft_Pane"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <action name="actionOpenImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSavePointsLeft">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionSavePointsRight">
   <property name="text">
    <string>Save Points</string>
   </property>
  </action>
  <action name="actionOpenImageRight">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionLoadPointsLeft">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionLoadPointsRight">
   <property name="text">
    <string>Load Points</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionOpenPointCloudLeft">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionOpenPointCloudRight">
   <property name="text">
    <string>Open PointCloud</string>
   </property>
  </action>
  <action name="actionFlipImage">
   <property name="text">
    <string>Flip Image</string>
   </property>
  </action>
  <action name="actionFlipRightImage">
   <property name="text">
    <string>Flip Right Image</string>
   </property>
  </action>
  <action name="actionSaveResult">
   <property name="text">
    <string>Save Result</string>
   </property>
  </action>
  <action name="actionOpenMask">
   <property name="text">
    <string>Open Mask</string>
   </property>
  </action>
  <action name="actionOpenMaskInverted">
   <property name="text">
    <string>Open Mask Inverted</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
  <customwidget>
   <class>PatchInfoWidget</class>
   <extends>QWidget</extends>
   <header>PatchInfoWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
